custom_components/healthchecksio/__init__.py

```
@@ -5,36 +5,48 @@
     ISSUE_URL,
     OFFICIAL_SITE_ROOT,
     REQUIRED_FILES,
 )
 
+_LOGGER = logging.getLogger(__name__)
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=300)
 
 
-async def async_setup(hass: core.HomeAssistant, config: ConfigType):
+async def async_setup(
+    hass: core.HomeAssistant, config: config_entries.ConfigEntry
```

Ruff (0.11.9)

Unused function argument: `hass`

(ARG001)